codegens/csharp-restsharp/lib/restsharp.js

```diff
@@ -17,7 +17,9 @@ function makeSnippet (request, options) {
     var snippet = `var client = new RestClient("${sanitize(request.url.toString())}");\n`,
         isUnSupportedMethod = UNSUPPORTED_METHODS_LIKE_GET.includes(request.method) ||
             UNSUPPORTED_METHODS_LIKE_POST.includes(request.method);
-
+    if (options.requestTimeout) {
```

Comment:
@shreys7 Can you confirm if not adding a timeout means the code waits indefinitely?

Reply:
@abhijitkane The default timeout is 100 seconds. Adding the timeout as -1 is considered as infinite timeout. Will make the requrired changes.